src/core/Akka.Cluster/SBR/SplitBrainResolver.cs

```diff
@@ -51,20 +42,23 @@ public static Props Props2(TimeSpan stableAfter, DowningStrategy strategy)
         // re-subscribe when restart
         protected override void PreStart()
         {
-            cluster.Subscribe(Self, InitialStateAsEvents, typeof(IClusterDomainEvent));
+            _cluster = Cluster.Get(Context.System);
```

Comment:
This fix was a lot simpler than I expected - just acquire the `Cluster` _after_ this actor has started, rather than doing it in the constructor.

Reply:
What it does is it avoids crashing the actor who spawned it, the `ClusterDaemon`, which is what was happening when this call was made in the constructor. The `PreStart` _does_ happen on a different run on the scheduler because it's part of the `Create` messaging-loop powered by the actors parent. It just all happens through `/system` messages that way.